ENGAGEHF/Dashboard/Greeting.swift

```diff
@@ -0,0 +1,30 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct Greeting: View {
+    var body: some View {
+        HStack(alignment: .top) {
+            Text("Hello, world!")
+                .font(.title.bold())
+            Spacer()
+            Text(.now, style: .date)
+                .font(.title3)
+                .foregroundStyle(.secondary)
+        }
+        .padding()
+    }
+}
+
+#if DEBUG
+#Preview {
+    Greeting()
+}
+#endif
```

Comment:
You can generally remove the #If DEBUG here is it is not really needed and while Previews will be accounted for in the test coverage this will make all of the logic a bit easier to see and understand without additional #if complexities. We had them in the template app for a while but started to phase this out in a few projects.

I would also always suggest two empty lines between type definitions (Greeting and the preview)
```suggestion

#Preview {
    Greeting()
}
```